assets/hip-1056/protobuf/stream/output/smart_contract_service.proto

```diff
@@ -0,0 +1,183 @@
+/**
+ * #  Service
+ * Block stream messages that report the results of transactions handled
+ * by the `smart contract` service.
+ *
+ * > REVIEW NOTE
+ * >> The use of sidecar records is a bit odd here. We may find it more
+ * >> effective to extract the actual changes into proper output messages
+ * >> and fields included in the ethereum call output and/or related state
+ * >> changes items, and remove the whole sidecar concept going forward.
+ *
+ * ### Keywords
+ * The key words "MUST", "MUST NOT", "REQUIRED", "SHALL", "SHALL NOT",
+ * "SHOULD", "SHOULD NOT", "RECOMMENDED", "MAY", and "OPTIONAL" in this
+ * document are to be interpreted as described in
+ * [RFC2119](https://www.ietf.org/rfc/rfc2119) and clarified in
+ * [RFC8174](https://www.ietf.org/rfc/rfc8174).
+ */
+syntax = "proto3";
+
+package com.hedera.hapi.block.stream.output;
+
+/*
+ * Copyright (C) 2024 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+option java_package = "com.hedera.hapi.block.stream.output.protoc";
+// <<<pbj.java_package = "com.hedera.hapi.block.stream.output">>> This comment is special code for setting PBJ Compiler java package
+option java_multiple_files = true;
+
+import "contract_call_local.proto";
+import "sidecar_file.proto";
+
+/**
+ * Block Stream data for a `contractCallMethod` transaction.
+ *
+ * This message SHALL NOT duplicate information already contained in
+ * the original transaction.
+ */
+message CallContractOutput {
+     /**
+     * A result for a Contract Call Transaction execution, either a create or call.
+     * <p>
+     * This field SHALL contain all of the data produced by the contract
+     * create transaction as well as basic accounting results.
+     */
```

Comment:
All of these modified or new comment blocks have incorrect indentation.
It might be worthwhile to modify whatever editor is in use to handle these correctly.

Reply:
Fixed like 90% of them. The remaing 10% are single spaces on the comments I believe

Reply:
I still see the alignment is off.  Should be one extra space indent in the comment body.
```suggestion
     /**
      * A result for a Contract Call Transaction execution, either a create or call.
      * <p>
      * This field SHALL contain all of the data produced by the contract
      * create transaction as well as basic accounting results.
      */
```

Reply:
Yep, that's the 10% single space I meant.
Will circle back